components/automate-cli/cmd/chef-automate/config.go

```diff
@@ -357,7 +358,9 @@ func runPatchCommand(cmd *cobra.Command, args []string) error {
 		}
 
 		configFile := args[0]
-		frontendCmd := fmt.Sprintf(FRONTEND_COMMAND, PATCH, "frontend"+"_"+timestamp+"_"+configFile, dateFormat)
```

Comment:
This line was the root cause of the issue, as the configFile = args[0] was appended to cmd, which later failed as relative path was different. when the config was located some other place.